explorer/explorerroutes.go

```diff
@@ -676,17 +686,20 @@ func (exp *explorerUI) Search(w http.ResponseWriter, r *http.Request) {
 		http.Redirect(w, r, "/tx/"+searchStr, http.StatusPermanentRedirect)
 		return
 	}
-	exp.ErrorPage(w, "search failed", "Could not find any transaction or block "+searchStr, true)
+	exp.StatusPage(w, "search failed", "Could not find any transaction or block "+searchStr, NotFoundStatusType)
 }
 
-// ErrorPage provides a way to show error on the pages without redirecting
-func (exp *explorerUI) ErrorPage(w http.ResponseWriter, code string, message string, notFound bool) {
-	str, err := exp.templates.execTemplateToString("error", struct {
-		ErrorCode   string
-		ErrorString string
-		Version     string
-		NetName     string
+// StatusPage provides a way to show the supported status on a pages
```

Comment:
...provides a page for displaying status messages and exception handling without redirecting.

Reply:
Fixed